### ISSUE TITLE
Bug: package.json contains invalid deps versions

### ISSUE THREAD

**Issue by antongolub** [qiwi/multi-semantic-release#27]
Hey @davikawasaki @oljekechoro, I just had a quick test with the new version and it seems the issue was not completely resolved. The beta suffix is now available but the version is incorrect. The screenshot to capture the issue is attached below but the details can be found [here](https://github.com/zlatanpham/monorepo-semantic-release-demo/pull/22/files).

![Screen Shot 2020-12-24 at 1 16 01 PM](https://user-images.githubusercontent.com/12707960/103065935-5bd0b380-45ea-11eb-87aa-807f8161af94.png)

_Originally posted by @zlatanpham in https://github.com/qiwi/multi-semantic-release/issues/25#issuecomment-750764078_

**Comment by antongolub** [comment]
@zlatanpham,

Looks very similar to https://github.com/dhoulb/multi-semantic-release/issues/47. At first we will try to localize the problem. Stay tuned.

**Comment by davikawasaki** [comment]
@antongolub don't know if you have fixed this yet, but it seems like [the way we bump local dependencies](https://github.com/qiwi/multi-semantic-release/blob/master/lib/updateDeps.js#L33) is different from the way we bump the package. Maybe we should standardize it... How are we bumping the main package (i.e. packages/apple) to 12.0.0-beta.1?

**Comment by antongolub** [comment]
I agree, there is a rather confusing piece. Any ideas how this could be improved?

**Comment by davikawasaki** [comment]
I'll tinker on this through these holidays days and I'll let you know.

**Comment by antongolub** [comment]
@zlatanpham,

[The fix](https://github.com/qiwi/multi-semantic-release/pull/34) is ready for testing (many thanks to @davikawasaki). Could you take a look and give feedback?

**Comment by Ellosaur** [comment]
@antongolub as we're using `"multi-semantic-release": "^2.5.3"` in our project can we simply hot swap to `"@qiwi/multi-semantic-release": "3.12.2"` or should we wait until the fix makes it upstream to `dhoulb/multi-semantic-release`?

**Comment by antongolub** [comment]
@Ellosaur,

As you wish. @qiwi-scoped msr is just like a canary build channel.

**Comment by Ellosaur** [comment]
So far so good! Our first release since switching has upgraded the changelog and dependencies correctly. Will keep an eye on it for a while as the problem was intermittent before. Thanks for the quick response!

**Comment by antongolub** [comment]
@Ellosaur,

Could you show `.releaserc`? I need to know what plugins are used to observe see how they modify `package.json` files.

**Comment by Ellosaur** [comment]
@antongolub,

Sure. We use the `release` section in the packages.json rather than a `.releaserc`. I included these on the original issue I logged [dhoulb#47](https://github.com/dhoulb/multi-semantic-release/issues/47). Those setting haven't changed.

**Comment by Ellosaur** [comment]
FYI we're now using [semantic-release-monorepo](https://github.com/pmowrer/semantic-release-monorepo) which doesn't attempt to update package references so it avoids this problem. We leave references as `~1.x` so that the latest package is always referenced
